docker/databend-query-docker.toml

```diff
@@ -23,6 +23,10 @@ mysql_handler_port = 3307
 clickhouse_handler_host = "0.0.0.0"
 clickhouse_handler_port = 9001
 
+# Databend Query ClickHouse Handler.
+clickhouse_http_handler_host = "0.0.0.0"
```

Comment:
How about providing a `clickhouse_http_handler_addr` instead?

Reply:
SGTM, but not quite sure.